src/main/java/core/basesyntax/db/FruitTransaction.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.db;
+
+public class FruitTransaction {
+    private String fruit;
+    private int quantity;
+    private String type;
+
+    public FruitTransaction(String fruit, int quantity) {
```

Comment:
The first constructor is missing the 'type' parameter, which seems to be a required field for a transaction. You should ensure that all necessary information for a transaction is initialized.